### ISSUE TITLE
"Explore" Button Labeled As "Create Chart" When Creating Virtual Dataset

### ISSUE THREAD

**Issue by lorho** [apache/superset#22802]
## Screenshot
![image](https://user-images.githubusercontent.com/123161754/213670329-bafc836d-385e-4ee9-a504-ddcf496c7c6b.png)

## Description

When trying to create a virtual dataset by using the SQL Editor, you write your query, then have to click on the "Explore" button to save it as a virtual dataset.
This button seems to have been renamed to "Create Chart", which doesn't fit what it does, since the popup that appears clearly states that you're creating a virtual dataset, not a chart (see screenshot).

## Design input
Please rename the button to either "Explore" or "Create Virtual Dataset", if this is supposed to be the way we create virtual datasets.

**Comment by rusackas** [comment]
Pinging @kasiazjc on the design/nomenclature front, and @eschutho on the engineering front, to help has this out. This might be better suited as a Discussion, but I'll leave that to you :D

**Comment by eschutho** [comment]
Hi @dwaux00 what version of the app are you on? Asking because in 2.0 this functionality was changed so that you no longer have to create a virtual dataset with this flow.

**Comment by dwaux00** [comment]
Hi @eschutho, thanks for your quick reply!
We're on 2.0.1:
![image](https://user-images.githubusercontent.com/123161754/214527815-c77fc2dc-0224-4ab3-ae61-31872ba26d25.png)
I thought this was the latest release version.

You're saying the flow changed: how? :) I'll gladly close this issue, if I got something wrong regarding creating a virtual dataset, in case the flow changed.

**Comment by rusackas** [comment]
Essentially, when you run a query in SQL Lab, you get the (renamed) "Create Chart" button. When you do this, you go _straight into_ the Explore tool (not creating a dataset). You can then fiddle with the knobs and buttons, and build a chart. When you go to save that chart, you're prompted to create a dataset. Creating the dataset is no longer a requisite intermediary step before creating a chart. 

If that all makes sense, I reckon we can probably close this, but if there's more worth discussing, we're game :D

**Comment by dwaux00** [comment]
After running my SQL query in the SQL Editor, then clicking on "Create Chart", I get the popup from my initial post. If I then choose "save as new", a virtual dataset gets be created. In your comment you wrote that I'd go straight into the Explore tool, which isn't the case for me. Only after clicking "Save & Explore" (which creates a virtual dataset) I get to the chart creation page.
To me this means the button doesn't create a chart, but a virtual dataset, just like it used to be - it just got renamed to something it doesn't actually do :D

In the chart creation (after clicking on "Save & Explore" in the popup) the new created dataset is automatically chosen:
![image](https://user-images.githubusercontent.com/123161754/214840250-536d08ee-8604-4d8c-a69c-6ccd3ae0310a.png)

So I'm actually still creating a dataset before creating the chart. 

I'm okay with closing this issue, but I definitely disagree with naming the button "Create Chart", when you're actually creating a dataset first. :D
Thank you for your help so far, really appreciate it!

**Comment by rusackas** [comment]
@dwaux00 what version of Superset are you running? Maybe I'm taking crazy pills, but I'm wondering if you're in some state of code between the renaming and the workflow change somehow.

**Comment by dwaux00** [comment]
We're on version 2.0.1:
https://user-images.githubusercontent.com/123161754/214527815-c77fc2dc-0224-4ab3-ae61-31872ba26d25.png

I can ping our devs and ask them to double check that they're actually using the latest version, but from what I've seen on the releases page, 2.0.1 is the latest (not counting the release candidate).
Other than that, we haven't done any changes, except for putting our company logo in the corner :)

**Comment by eschutho** [comment]
Ok, correct, in 2.0.1 the new functionality hasn't been released yet where you wouldn't have to create a dataset. So the state of this is a little odd here. We can either rename the button for 2.0.2 or wait until the next set of features are released in 2.1 or 3.0 which will actually take you to a chart.

**Comment by rusackas** [comment]
This is likely fixed by now, and is pretty out of date if not. If people are still encountering this in current versions (3.x) please open a new Issue or a PR to address the problem.
